src/userscores/CMakeLists.txt

```diff
@@ -41,6 +41,8 @@ set(MODULE_SRC
     ${CMAKE_CURRENT_LIST_DIR}/view/templatesmodel.h
     ${CMAKE_CURRENT_LIST_DIR}/view/templatepaintview.cpp
     ${CMAKE_CURRENT_LIST_DIR}/view/templatepaintview.h
+    ${CMAKE_CURRENT_LIST_DIR}/view/exportscoremodel.h
```

Comment:
please replace the .h and .cpp
it is code-style

Reply:
So, always first .cpp, _then_ .h:
```cmake
${CMAKE_CURRENT_LIST_DIR}/view/exportscoremodel.cpp
${CMAKE_CURRENT_LIST_DIR}/view/exportscoremodel.h
``` 
etc.